New-language-adaptation/German/training/1-train-acoustic-model.ipynb

```diff
@@ -0,0 +1,553 @@
+     ]
+    }
+   ],
+   "source": [
+    "print(asr_model.decoder.vocabulary)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "5BBtk30g5sHJ"
+   },
+   "source": [
+    "Now let's update the vocabulary in this model, using the German tokenizer that we have trained in the data preparation step."
```

Comment:
```suggestion
    "Now, let's update the vocabulary in this model, using the German tokenizer that we have trained in the data preparation step."
```

Reply:
just a nit but when we refer to a step by title, like data preparation step, i'm assuming we've used the term data preparation so i make sure its a step that i completed.  i don't know what we are referring to here when we say data preparation step.